include/wil/chrono.h

```diff
@@ -0,0 +1,370 @@
+//*********************************************************
+//
+//    Copyright (c) Microsoft. All rights reserved.
+//    This code is licensed under the MIT License.
+//    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF
+//    ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+//    TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+//    PARTICULAR PURPOSE AND NONINFRINGEMENT.
+//
+//*********************************************************
+#ifndef __WIL_CHRONO_INCLUDED
+#define __WIL_CHRONO_INCLUDED
+
+#include "common.h"
+#include "result.h"
+
+#ifdef WIL_KERNEL_MODE
+#error This header is not supported in kernel-mode.
+#endif
+
+#if __WI_CPLUSPLUS < 201103L
+#error This header requires C++11 or later
+#endif
+
+#include <chrono>
+
+#include <minwindef.h> // FILETIME, etc.
+#include <sysinfoapi.h> // GetTickCount[64], GetSystemTime[Precise]AsFileTime
+#include <realtimeapiset.h> // Query[Unbiased]InterruptTime[Precise]
+
+namespace wil
+{
+#pragma region std::chrono wrappers for GetTickCount[64]
+    namespace details
+    {
+        template<typename rep_t, typename get_tick_count_fn_t, get_tick_count_fn_t get_tick_count_fn, typename base_clock_t>
+        struct tick_count_clock_impl
+        {
+            using rep = rep_t;
+            using period = std::milli;
+            using duration = std::chrono::duration<rep, period>;
+            using time_point = std::chrono::time_point<base_clock_t, duration>;
+            static constexpr bool const is_steady = true;

```

Comment:
I think for integral type `constexpr` with `const` together is useless, so we can remove it.

Reply:
Done